### ISSUE TITLE
Chunk Steam for World 

### ISSUE THREAD

**Issue by MineSunshineone** [SuCraft/Suki#2]
Briefly describe:
I fork the lastset version ---commit "423d985351c67c3aff16fea023adccd57a97c217" 
and i got it 
[crash-2022-10-13_09.42.54-server.txt](https://github.com/SuCraft/Suki/files/9769712/crash-2022-10-13_09.42.54-server.txt)

but When I switched to this version ---git-Suki-"88eb346" ---commit "44ff5c15fecf6a3b21f96d1cc3dad9b9d4344d19", everything was back to normal

Plugins (19): 
AnCDK, AnvilColor, ChestProtect, Chunky, CoreProtect, CreeperConfetti, DeathMessage*, Essentials, EssentialsChat, GravityControl, GSit, InvSeePlusPlus, LuckPerms, MiraiMC, PistonMOTD, spark, TAB, Vault, Whitelist4QQ

Also There's still a small request here
If you can add simulation-distance back and add Tpsbar and my favorite purpur in Sand-fix and end_portal: safe-teleporting: false Suki would be a really good fork

Finally any more info you need ,plz ping me : )

**Comment by MartijnMuijsers** [comment]
Thank you for the issue :)

- [x] The error you reported
- [ ] Add simulation-distance back
- [ ] Add Tpsbar
- [ ] Sand-fix
- [ ] end_portal: safe-teleporting: false

For *simulation-distance*, what do you mean with adding it back?

**Comment by MineSunshineone** [comment]
it doesnt work

**Comment by MartijnMuijsers** [comment]
> it doesnt work

Do you mean the `server.properties` setting? If yes, what doesn't work about it?

**Comment by MineSunshineone** [comment]
Yes,i dont know why it doesnt work, Suki only identify Tick-chunk in server.properties

**Comment by MartijnMuijsers** [comment]
*What* doesn't work?

**Comment by MineSunshineone** [comment]
simulation-distance

**Comment by MartijnMuijsers** [comment]
What do you expect, and what happens instead?

**Comment by MartijnMuijsers** [comment]
（更容易的话你也可以用中文了解）

**Comment by MineSunshineone** [comment]
你能看懂吗

**Comment by MartijnMuijsers** [comment]
我会，会说中文 ；p

**Comment by MineSunshineone** [comment]
怎么说呢，Suki不识别模拟视距，只能识别Tick-Chunk

**Comment by MineSunshineone** [comment]
但是你调大了Tick-chunk 感觉性能又不咋样了，我也尝试过去更改后的paper-config里面改track，但是好像也没啥用

**Comment by MartijnMuijsers** [comment]
Tick-chunk是什么意思？Tick distance和simulation distance是一样的

**Comment by MartijnMuijsers** [comment]
Sand duping fix configuration has been added to Gale: https://github.com/GaleMC/Gale/blob/master/patches/server/0055-Make-sand-duping-fix-configurable.patch and will later be pulled into Suki
